src/chainhook/server.ts

```diff
@@ -125,6 +130,7 @@ const Chainhook: FastifyPluginCallback<Record<never, never>, Server, TypeBoxType
   fastify.post('/chainhook/inscription_revealed', async (request, reply) => {
     try {
       await processInscriptionRevealed(request.body, fastify.db);
+      refreshMaterializedViewsDebounced(fastify.db);
```

Comment:
Chainhook messages come once per block, so it's not necessary to create this debounced logic. You could simply issue a refresh once per message.